.travis.yml

```diff
@@ -3,7 +3,7 @@ dist: trusty
 language: generic
 install: 
  - git clone https://github.com/QubesOS/qubes-builder ~/qubes-builder
- - wget http://i3wm.org/downloads/i3-4.12.tar.bz2 -O i3/i3-4.12.tar.bz2
+ - wget https://i3wm.org/downloads/i3-$PACKAGE_VERSION.tar.bz2 -O i3/i3-$PACKAGE_VERSION.tar.bz2
```

Comment:
In fact, this wget isn't needed here at all. `travis-build` script do handle it already. I'll remove this part after merging.